### PR TITLE
quick fix : set language id for issue #99

### PR DESCRIPTION
**Pull request by mhidaka**
fixed https://github.com/konifar/droidkaigi2016/issues/99

### CODE REVIEW

File: app/src/main/res/raw/sessions_ja.json [hunk 1]
```diff
@@ -812,7 +812,7 @@
       "id": 4,
       "name": "ルームC"
     },
-    "language_id": "",
+    "language_id": "ja",
```
Comment (hotchemi) [inline app/src/main/res/raw/sessions_ja.json]: This is @konifar 's talk right?

**Comment by konifar** [comment]
Perfect fix! Thanks, well done sir.

**Comment by konifar** [comment]
LGTM

**Comment by KeithYokoma** [comment]
よさ

File: app/src/main/res/raw/sessions_ja.json (hunk 1 above)
Comment (konifar) [inline app/src/main/res/raw/sessions_ja.json]: Yes, I speak in Japanese. But I am going to try to speech in English at only first greeting.

**Comment by hkurokawa** [comment]
LGTM. @mhidaka Could you merge it by yourself?  Thanks.

**Comment by mhidaka** [comment]
Thanks Guys

**Pull request merged**